recipes/assimp/5.x/conanfile.py

```diff
@@ -128,6 +128,12 @@ def _depends_on_poly2tri(self):
     def _depends_on_rapidjson(self):
         return self.options.with_gltf or self.options.with_gltf_exporter
 
+    @property
+    def _depends_on_draco(self):
+        if tools.Version(self.version) < tools.Version("5.1.0"):
```

Comment:
Is the double version required? I've never see that before 🤔

Reply:
Ah, yes, it is not needed indeed, but should work fine. I probably copied it from somewhere without thinking, my bad.